fast_faults.py

```diff
@@ -27,7 +19,6 @@ def __init__(self, channel=None, *args, **kwargs):
         self.channel = None
         if channel:
             self.channel = PyDMChannel(channel, connection_slot=self.connection_changed)
-        self.destroyed.connect(functools.partial(clear_channel, channel))
```

Comment:
Removed but not addressed in a different way? Is clean-up no longer a concern here?

Reply:
With pcds-5.8.0, the only impact of this line's clean-up (and similar cleanups) is to cause the various "channel already removed" error spam race conditions. PyDM adequately cleans up its own channels and pyepics adequately cleans up its own EPICS resources at exit.

I assume the intent of this line was related to widgets getting deleted without cleaning up the associated channels, but that doesn't happen in this application- perhaps because widget lifetimes are all the way to the end of the program? Maybe it's related to PyDM finding the `channels` method? I'm not fully sure.

Reply:
So long as these widgets never leave `pmps-ui` with the intent of being reused, leaving clean-up to exit is just fine, I think

Reply:
Admittedly, there's probably a way to fix these so that they don't cause issues rather than completely killing them, but I wasn't able to quickly identify it.